### PR TITLE
forgejo: new, 7.0.5

### PR DESCRIPTION
**Pull request by stdmnpkg**
Topic Description
-----------------

- forgejo: bump REL for topic Revision Marking Guidelines
- forgejo: new, 7.0.5

Package(s) Affected
-------------------

- forgejo: 7.0.5-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit forgejo
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Comment by stdmnpkg** [comment]
works on my x86 virtual machine

image

File: app-web/forgejo/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,7 @@
+abinfo "Building Forgejo"
+unset LDFLAGS
+TAGS="bindata timetzdata sqlite sqlite_unlock_notify" make build
+
+abinfo "Installing Forgejo ..."
+install -Dvm755 $SRCDIR/gitea $PKGDIR/usr/bin/forgejo
```
Comment (MingcongBai) [inline app-web/forgejo/autobuild/build]: Quote both path variables.

File: app-web/forgejo/autobuild/build [hunk 2]
```diff
@@ -0,0 +1,7 @@
+abinfo "Building Forgejo"
+unset LDFLAGS
```
Comment (MingcongBai) [inline app-web/forgejo/autobuild/build:3]: Why?

File: app-web/forgejo/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+PKGSEC=web
+PKGDES="A self-hosted lightweight software forge"
+PKGDEP="glibc git"
+PKGRECOM="openssh"
+
+BUILDDEP="go nodejs"
+ABSPLITDBG=0
+
+# FIXME: runtime.cgo_yield: relocation target _cgo_yield not defined
+FAIL_ARCH=riscv64
```
Comment (MingcongBai) [inline app-web/forgejo/autobuild/defines]: Quote FAIL_ARCH values, also missing a newline at end-of-file (VSCode?).

File: app-web/forgejo/autobuild/defines [hunk 4]
```diff
@@ -0,0 +1,11 @@
+PKGNAME=forgejo
+PKGSEC=web
+PKGDES="A self-hosted lightweight software forge"
```
Comment (MingcongBai) [inline app-web/forgejo/autobuild/defines]: A code hosting platform for collaborative development

File: app-web/forgejo/autobuild/defines [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+PKGDEP="glibc git"
+PKGRECOM="openssh"
+
+BUILDDEP="go nodejs"
+ABSPLITDBG=0
```
Comment (MingcongBai) [inline app-web/forgejo/autobuild/defines:10]: Please add:

```
# FIXME: Autobuild does not yet support splitting out debug symbol from Go executables.
```

File: app-web/forgejo/autobuild/overrides/usr/lib/systemd/system/forgejo.service [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+ProtectKernelTunables=true
+ProtectProc=invisible
+ProtectSystem=strict
+RestrictAddressFamilies=AF_INET AF_INET6 AF_NETLINK AF_UNIX
+RestrictNamespaces=true
+RestrictRealtime=true
+RestrictSUIDSGID=true
+SystemCallArchitectures=native
+SystemCallFilter=@system-service
+SystemCallErrorNumber=EPERM
```
Comment (MingcongBai) [inline app-web/forgejo/autobuild/overrides/usr/lib/systemd/system/forgejo.service]: Holy smokes. Where did all these come from?

File: app-web/forgejo/autobuild/postinst [hunk 7]
```diff
@@ -0,0 +1,5 @@
+echo "Creating user for forgejo ..."
```
Comment (MingcongBai) [inline app-web/forgejo/autobuild/postinst]: Please change "user" to "daemon owner"

File: app-web/forgejo/spec [hunk 8]
```diff
@@ -0,0 +1,5 @@
+VER=7.0.5
+# copy-repo required by auto version detection
```
Comment (MingcongBai) [inline app-web/forgejo/spec]: Change "#" to "# Note:" to make comments easier to distinguish.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-web/forgejo/autobuild/build (hunk 2 above)
Comment (stdmnpkg) [reply app-web/forgejo/autobuild/build:3]: does not compile with our default LDFLAGS, added a comment to explain this.

File: app-web/forgejo/autobuild/overrides/usr/lib/systemd/system/forgejo.service (hunk 6 above)
Comment (stdmnpkg) [reply app-web/forgejo/autobuild/overrides/usr/lib/systemd/system/forgejo.service]: copied from Archlinux, should i remove these?

Comment (MingcongBai) [reply app-web/forgejo/autobuild/overrides/usr/lib/systemd/system/forgejo.service]: No, but would be good to note where they came from (and whether we could send this unit file upstream).

Comment (stdmnpkg) [reply app-web/forgejo/autobuild/overrides/usr/lib/systemd/system/forgejo.service]: replaced with patched upstream provided service file, didn't realize they provided one before

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**